apps/wallet/src/ui/app/ApiProvider.ts

```diff
@@ -103,14 +91,15 @@ export default class ApiProvider {
             this.setNewJsonRpcProvider();
         }
         if (!this._signer) {
-            this._signer = growthbook.isOn(FEATURES.DEPRECATE_GATEWAY)
-                ? new RawSigner(
-                      keypair,
-                      this._apiFullNodeProvider,
-                      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            this._signer = new RawSigner(
+                keypair,
+                this._apiFullNodeProvider,
+
+                growthbook.isOn(FEATURES.USE_LOCAL_TXN_SERIALIZER)
```

Comment:
There's some gap in the local txn serializer, so adding this flag to switch between the RPC txn builder and the local one